nipype/pipeline/plugins/multiproc.py

```diff
@@ -74,25 +75,6 @@ def run_node(node, updatehash, taskid):
     return result
 
 
-class NonDaemonProcess(Process):
-    """A non-daemon process to support internal multiprocessing.
-    """
-
-    def _get_daemon(self):
-        return False
-
-    def _set_daemon(self, value):
-        pass
-
-    daemon = property(_get_daemon, _set_daemon)
-
-
-class NonDaemonPool(pool.Pool):
```

Comment:
I think we still want nondaemon processes, we will need to investigate how to do that with concurrent.futures

Reply:
It will require using undocumented API, if it is possible.

What purpose does it serve, by the way?